### ISSUE TITLE
This project should be reference from grails.org

### ISSUE THREAD

**Issue by ansoni** [beckje01/grails-spring-security-saml#5]
fpanwaskar/grails-spring-security-saml project hasn't been maintained in a few years.  Between you keeping this up to date and your slides.com documentation effort, it feels like this should be the authoritative plugin.

**Comment by beckje01** [comment]
I would like to make that a happen after having this updated and running in production with the Spring Security SAML 1.0

**Comment by virtualdogbert** [comment]
I agree with ansoni, I'm just starting to research SAML, and other security aspects beyond core, and seeing that the current "authoritative" plugin, wasn't maintained, didn't inspire a lot of confidence. I'm glad I dug a little deep and found your branch.

You say that you're working on the upgrade to to the newest spring security, does that mean that the current code doesn't work with spring security core 2.0-RC4?

**Comment by beckje01** [comment]
I haven't fully tested it in production with core 2.0

But pull requests are welcome if you do some testing and find issues.

Jeff

**Comment by ansoni** [comment]
I will have some pull requests coming your way for core 2.0.

I currently have it accepting a SAML assertion and setting roles properly, but dealing with a permission denied even thought I am authenticated and have authorities.
